ipi/pes/fileIOCalculator.py

```diff
@@ -0,0 +1,334 @@
+import os
+import time
+import json
+import shutil
+import re
+import numpy as np
+from datetime import datetime
+from filelock import FileLock
+from ase import Atoms
+from ase.io import write
+from ase.calculators.calculator import Calculator, all_changes
+from typing import Dict, Union, Any
+from .ase import ASEDriver
+
```

Comment:
here make sure to use as few modules as you can in order to decrease dependencies and make code more portable.